resources/frontend.rb

```diff
@@ -19,8 +19,18 @@
       variables['frontend'][new_resource.name] ||= {}
       variables['frontend'][new_resource.name]['default_backend'] ||= ''
       variables['frontend'][new_resource.name]['default_backend'] << new_resource.default_backend
-      variables['frontend'][new_resource.name]['bind'] ||= ''
-      variables['frontend'][new_resource.name]['bind'] << new_resource.bind
+      variables['frontend'][new_resource.name]['bind'] = []
+      if new_resource.bind.is_a? Hash
+        new_resource.bind.map do |addresses, ports|
+          addresses = Array(addresses) unless addresses.is_a? Array
+          ports = Array(ports) unless ports.is_a? Array
```

Comment:
I think it's a remnant of a test done in the past. I think it can actually be removed.